integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/ChromeWebDriverProvider.java

```diff
@@ -103,19 +104,34 @@ private void downloadZipFile(String url, String zipFileName, String fileName) {
       return;
     }
 
-    try {
-      LOG.info("Download the zip file from " + url + " to " + downLoadDir);
-      File chromeDriverZip = new File(ChromeWebDriverProvider.downLoadDir, zipFileName);
-      FileUtils.copyURLToFile(new URL(url), chromeDriverZip);
+    Instant limit = Instant.now().plusSeconds(120);
+    int retryNum = 0;
+    IOException last = null;
 
-      LOG.info("Extract the zip file from " + chromeDriverZip.getAbsolutePath());
-      Archiver archiver = ArchiverFactory.createArchiver(ArchiveFormat.ZIP);
-      archiver.extract(new File(downLoadDir, zipFileName), new File(downLoadDir));
-    } catch (IOException e) {
-      LOG.error(
-          "Download of: " + url + ", failed in path " + ChromeWebDriverProvider.downLoadDir, e);
-      throw new RuntimeException(e);
+    while (retryNum < 3 && Instant.now().isBefore(limit)) {
+      try {
+        LOG.info("Download the zip file from " + url + " to " + downLoadDir);
+        File chromeDriverZip = new File(ChromeWebDriverProvider.downLoadDir, zipFileName);
+        if (chromeDriverZip.exists()) {
+          chromeDriverZip.delete();
+        }
```

Comment:
Maybe we don't have to delete it when existed.

Reply:
I would suggest to rename a unique temporary name for download and unzip, when successfully downloaded and unzipped, then renaming to the target name?

Reply:
ok

Reply:
done